content/sql/concepts/operators/terms/null-safe-equal/null-safe-equal.md

```diff
@@ -0,0 +1,40 @@
+---
+Title: 'NULL SAFE EQUAL'
+Description: 'Performs an inequality comparison similar to the '=' operator, using the logic of an XNOR gate. Can accept NULL as a value and will not output NULL.'
+Subjects: 
+  - 'Computer Science'
+Tags: 
+  - 'Operators'
+  - 'Database'
+  - 'Queries'
+  - 'Tables'
```

Comment:
```suggestion
Tags: 
  - 'Operators'
  - 'Database'
  - 'Queries'
  - 'Tables'
```
Include only those tags that are already present in the [tags.md list ](documentation/tags.md)and you can add one if you want to include it in your PR. Here `operators` is not part of the tags list so you can add it if you want it to be used.